content/en/monitors/create/types/ci_pipelines.md

```diff
@@ -0,0 +1,119 @@
+    * **Monitor over a CI Pipeline event count**: Use the search bar (optional) and do **not** select a facet or measure. Datadog evaluates the number of CI Pipeline events over a selected time frame, then compares it to the threshold conditions.
+    * **Monitor over a facet**: If a facet is selected, the monitor alerts over the `Unique value count` of the facet.
+    * **Monitor over measure**: If a measure is selected, the monitor alerts over the numerical value of the CI Pipeline facet (similar to a metric monitor) and aggregation needs to be selected (`min`, `avg`, `sum`, `median`, `pc75`, `pc90`, `pc95`, `pc98`, `pc99`, or `max`).
+3. Group CI Pipeline events by multiple dimensions (optional):
+    All CI Pipeline events matching the query are aggregated into groups based on the value of up to four facets.
+4. Configure the alerting grouping strategy (optional):
+    * If the query has a `group by`, multi alerts apply the alert to each source according to your group parameters. An alerting event is generated for each group that meets the set conditions. For example, you could group a query by `@ci.pipeline.name` to receive a separate alert for each CI Pipeline name when the number of errors is high.
+
+{{< img src="monitors/monitor_types/ci_pipelines/define-the-search-query.png" alt="Define the search query" style="width:80%;" >}}
+
+#### Using Formulas & Functions
+
+You can create CI Pipeline monitors based on Formulas & Functions. This is useful if you want to trigger alerts for monitors based on the error rate for some pipeline.
+
+In the following example, we're creating an error rate monitor using the formula that calculates the ratio of `Failed CI Pipelines` / `Total CI Pipelines` grouped by `ci.pipeline.name`.
+
+{{< img src="monitors/monitor_types/ci_pipelines/define-the-search-query-fnf.png" alt="Define the search query" style="width:80%;" >}}
+
+<div class="alert alert-info"><strong>Note</strong>: There is a limit of 2 in the amount of queries that can be used to build the evaluation formula per monitor.</div>
+
+### Set alert conditions
+
+* Trigger when the metric is `above`, `above or equal to`, `below`, or `below or equal to`
+* the threshold during the last `5 minutes`, `15 minutes`, `1 hour`, etc. or `custom` to set a value between `1 minute` and `2 days`.
+* Alert threshold `<NUMBER>`
+* Warning threshold `<NUMBER>`
+
+#### Advanced alert conditions
+
+For detailed instructions on the advanced alert options (evaluation delay, etc.), see the [Monitor configuration][3] page.
+
+### Notifications
+
+For detailed instructions on the **Say what's happening** and **Notify your team** sections, see the [Notifications][4] page.
+
+#### Use event attributes in the message
+
+You can use all the attributes stored in a CI Pipeline event in your notification message using the `{{cipipeline.attributes.<attribute>}}` syntax. You can use all the CI Pipeline facets to build your custom notification message.
```

Comment:
I'd say this is not clear which event we're referring to. Is this a random event that matches? The event that made the transition happen? This is not explicit, and as a monitor could refer to several values for a single tag (such as `git.branch` if it is not specified in the monitor query).
Also, what does "all attributes" mean?

Reply:
Fragment removed as these instructions are already covered [here](https://docs-staging.datadoghq.com/drodriguezhdez/cipipelines-monitors_public_docs/monitors/notify/variables/?tab=is_alert#matching-attributetag-variables).